packager/util/config.go

```diff
@@ -48,6 +48,7 @@ type URLPattern struct {
 	SamePath               *bool
 }
 
+var emptyRegexp = ""
 var dotStarRegexp = ".*"
```

Comment:
any reason to keep `dotStarRegexp`?

Reply:
Yeah, still used above for PathRE. (For now.)